data/callToAction.json

```
@@ -4,7 +4,7 @@
         "link": "http://eepurl.com/hdjJMz"
     },
     {
-        "text": "Check out our Game Jam event",
-        "link": "https://gamejam.hackutd.co/"
+        "text": "Check out our Hackathon event",
```

Let's update the text to be more straightforward:
```suggestion
        "text": "Register for HackUTD VII: The VII Seas",
```